src/utils/feature-flags.mjs

```diff
@@ -0,0 +1,21 @@
+/**
+ * Allows us to check if a feature flag is enabled for a site.
+ * Due to versioning of the cli, and the desire to remove flags from
+ * our feature flag service when they should always evaluate to true,
+ * we can't just look for the presense of {featureFlagName: true}, as
+ * the absense of a flag should also evaluate to the flag being enabled.
+ * Instead, we return that the feature flag is enabled if it isn't
+ * specifically set to false in the response
+ * @param {import('netlify').NetlifyAPI} api
+ * @param {*} siteInfo
+ * @param {string} flagName
+ *
+ * @returns {boolean}
+ */
+export const isFeatureFlagEnabled = async (api, siteId, flagName) => {
```

Comment:
nit: Really down to personal preference, but I think having the flag name as the first parameter would make the code a bit easier to scan, since the name of the flag is probably the most important parameter and the other two are basically supporting metadata.

This is also in line with other feature flag interfaces across our codebases.